packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/RecentSubjectsContainer.js

```diff
@@ -0,0 +1,78 @@
+import asyncStates from '@zooniverse/async-states'
+import counterpart from 'counterpart'
+import { inject, observer } from 'mobx-react'
+import { bool } from 'prop-types'
+import React, { Component } from 'react'
+
+import RecentSubjects from './RecentSubjects'
+import RecentSubjectsCarousel from './RecentSubjectsCarousel'
+import MessageBox from './components/MessageBox'
+import Placeholder from './components/Placeholder'
+import fetchRecentSubjects from './helpers/fetchRecentSubjects'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper (stores) {
+  return {
+    projectId: stores.store.project.id,
+    slug: stores.store.project.slug
+  }
+}
+
+@inject(storeMapper)
+@observer
+class RecentSubjectsContainer extends Component {
+  constructor () {
+    super()
+    this.state = {
+      loading: asyncStates.initialized,
+      subjects: []
+    }
+  }
+
+  componentDidMount () {
+    this.setState({ loading: asyncStates.loading })
+    fetchRecentSubjects(this.props.projectId)
+      .then(subjects => this.setState({
+        loading: asyncStates.success,
+        subjects: subjects || []
+      }))
+      .catch(error => {
+        console.error(error)
+        this.setState({
+          loading: asyncStates.error
+        })
+      })
+  }
+
+  render () {
+    const { carousel } = this.props
+    const { loading, subjects } = this.state
+    const href=`/projects/${this.props.slug}/talk`
+    let result = null
+    const ThumbnailComponent = carousel ? RecentSubjectsCarousel : RecentSubjects
+
+    if (loading === asyncStates.success) {
+      result = (subjects.length < 1)
+        ? (<MessageBox children={counterpart('RecentSubjects.noSubjects')} />)
```

Comment:
The children message here should probably be wrapped by `Paragraph` or something to give it some padding. It looks a bit odd on smaller sizes without it:

iPad Pro size:
<img width="953" alt="Screen Shot 2019-08-30 at 12 07 45 PM" src="https://user-images.githubusercontent.com/5016731/64038912-ff0cd800-cb1e-11e9-982e-e7833d874ee1.png">

Mobile:
<img width="369" alt="Screen Shot 2019-08-30 at 12 07 52 PM" src="https://user-images.githubusercontent.com/5016731/64038919-0633e600-cb1f-11e9-9ef5-7540d6abed70.png">

Reply:
Fixed.